manifests/Mozilla/Thunderbird/68.8.0.yaml

```diff
@@ -0,0 +1,14 @@
+Id: Mozilla.Thunderbird
+Version: 68.8.0
+Name: Thunderbird
+Publisher: Mozilla
+License: Mozilla Public License 2.0 (MPL)
+LicenseUrl: https://www.mozilla.org/en-US/MPL/2.0/
+Tags: email
+Description: Thunderbird is a free email application that’s easy to set up and customize!
+Homepage: https://www.thunderbird.net/en-US/
+Installers:
+  - Arch: x64
+    Url: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/68.8.0/win32/en-US/Thunderbird%20Setup%2068.8.0.exe

```

Comment:
Since this is clearly intended for x64-PCs, I guess it would be better to also use the x64-version of thunderbird. 

This would be available here: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/68.8.0/win64/en-US/Thunderbird%20Setup%2068.8.0.exe

Reply:
you are totally right, my mistake!